tests/backend/test_GameManager.py

```diff
@@ -3,6 +3,47 @@
 from backend.GameManager import *
 
 
+def place_test_pieces(game_manager):
+    # Test piece placement
+    game_manager.place_piece(0, 0)
+    game_manager.end_turn()
+    game_manager.place_piece(0, 6)
+    game_manager.end_turn()
+    game_manager.place_piece(3, 0)
+    game_manager.end_turn()
+    game_manager.place_piece(3, 6)
+    game_manager.end_turn()
+    game_manager.place_piece(6, 0)
+    game_manager.end_turn()
+    game_manager.place_piece(6, 3)
+    game_manager.end_turn()
+    game_manager.place_piece(6, 6)
+    game_manager.end_turn()
+    game_manager.place_piece(1, 1)
+    game_manager.end_turn()
+    game_manager.place_piece(1, 3)
+    game_manager.end_turn()
+    game_manager.place_piece(1, 5)
+    game_manager.end_turn()
+    game_manager.place_piece(3, 1)
+    game_manager.end_turn()
+    game_manager.place_piece(3, 5)
+    game_manager.end_turn()
+    game_manager.place_piece(5, 1)
+    game_manager.end_turn()
+    game_manager.place_piece(5, 3)
+    game_manager.end_turn()
+    game_manager.place_piece(5, 5)
+    game_manager.end_turn()
+    game_manager.place_piece(2, 2)
+    game_manager.end_turn()
+    game_manager.place_piece(2, 3)
+    game_manager.end_turn()
+    game_manager.place_piece(2, 4)
+    game_manager.end_turn()
```

Comment:
Doesn't really matter here, but just so you guys know: the turn doesn't have to end after every placement, movement, etc. 

All pieces of one color can be placed, the turn can end (to swap colors), all pieces of the other type can be placed, and then the board can be used.